README.md

```diff
@@ -600,9 +600,12 @@ spec:
 
 ## Backup validation using a Policy
 
-The Cluster Back up and Restore Operator [helm chart](https://github.com/stolostron/cluster-backup-chart) installs the [backup-restore-enabled](https://github.com/stolostron/cluster-backup-chart/blob/main/stable/cluster-backup-chart/templates/hub-backup-pod.yaml) Policy, used to inform on issues with the backup and restore component. 
+The Cluster Backup and Restore Operator [helm chart](https://github.com/stolostron/multiclusterhub-operator/tree/main/pkg/templates/charts/toggle/cluster-backup) installs [backup-restore-enabled](https://github.com/stolostron/multiclusterhub-operator/blob/main/pkg/templates/charts/toggle/cluster-backup/templates/hub-backup-pod.yaml) and [backup-restore-auto-import](https://github.com/stolostron/multiclusterhub-operator/blob/main/pkg/templates/charts/toggle/cluster-backup/templates/hub-backup-auto-import.yaml) Policies, designed to provide information on issues with the backup and restore component. 
 
-The Policy has a set of templates which check for the following constraints and informs when any of them are violated. 
+These policies include a set of templates that check for the following constraints and alerts when any of them are violated.
+
+### OADP channel validation
+- `oadp-channel` template checks if the Red Hat OADP Operator is installed from the expected channel. The OADP operator is installed by MCH (Multicluster Hub) through the Cluster Backup and Restore Operator Helm chart, sourced from the latest supported OADP channel. OADP/Velero Custom Resource Definitions (CRDs) are cluster-scoped. If multiple versions of OADP are installed on the ACM (Advanced Cluster Management) hub, or if the version installed by MCH has been uninstalled and a different version is installed manually, the Backup and Restore Operator might run with incorrect CRDs, resulting in erroneous behavior.
```

Comment:
Can we have something like this

- `oadp-channel` template checks if the Red Hat OADP Operator is installed from the expected channel.
When you enable the backup component on the MCH (Multicluster Hub), the OADP operator is installed by the Cluster Backup and Restore Operator Helm chart. The oadp-channel template checks if the installed Red Hat OADP Operator version matches the version set by the ACM Cluster Backup and Restore Operator. The template shows violations if a Red Hat OADP Operator version is found to be installed on the hub and it doesn't match the version installed by the ACM Cluster Backup and Restore Operator Helm chart. The reason for showing this violation is that the OADP/Velero Custom Resource Definitions (CRDs) are cluster-scoped. If multiple versions of OADP are installed on the ACM (Advanced Cluster Management) hub, or if the version installed by MCH has been uninstalled and a different version is installed manually, the Backup and Restore Operator might run with incorrect CRDs, resulting in erroneous behavior.